### PR TITLE
change into new VAT for scheduled courses

### LINKED ISSUE #237: VAT Increase on Scheduled Courses
_From @christodewit on April 6, 2018 12:26_

# Problem
Incorrect VAT displayed on the website

# Solution
Increase the VAT to 15% as per SA VAT Increase

@ann26 @sonlinux @gubuntu 

_Copied from original issue: kartoza/Admin#215_

### PR DESCRIPTION
**Pull request by boney-bun**
now: 15pct, before: 14pct

fix #237

### CODE REVIEW

**Comment by meomancer** [comment]
hi @boney-bun 
- [x] as per my comment, we need to add it to admin site, so it can be changed through admin

- [x] and also
```
maybe we can linked vat value to the payment, so if someone want to
download or checking their old payment, the vat that show is old vat one
(we need to add data migrations in the process)
```
just add new field VAT in payment model, or maybe you could create new model to make relationship with vat model and payment
and use this model when user download pdf

- [ ] and also make migrations data to all current payment, to use old vat

**Comment by boney-bun** [comment]
the latest PR address the old vat in the old transaction.

Here is the sample:
newvat

File: django_project/config/defaults.py [hunk 1]
```diff
@@ -19,3 +19,27 @@
...
+
+register_setting(
+    name="CURRENT_VAT",
+    label="Current VAT",
+    description="The current VAT.",
+    editable=True,
+    default=15,
+)
+
+register_setting(
```
Comment (meomancer) [inline django_project/config/defaults.py:31]: hi @boney-bun 
is this OLD_VAT filled everytime we change CURRENT_VAT?

File: django_project/config/defaults.py [hunk 2]
```diff
@@ -19,3 +19,27 @@
...
+    name="OLD_VAT",
+    label="Old VAT",
+    description="The old VAT.",
+    editable=True,
+    default=14,
+)
+
+register_setting(
```
Comment (meomancer) [inline django_project/config/defaults.py]: hi @boney-bun 
the question same with here

File: django_project/config/defaults.py (hunk 1 above)
Comment (boney-bun) [reply django_project/config/defaults.py:31]: thanks for the comments @meomancer 

a user should manually change the values of `OLD_VAT`, `CURRENT_VAT` and `CURRENT_VAT_STARTED`. 
I think the VAT will not frequently change so probably it's better to keep it simple.
if we need to keep the history of VAT changes, then I may need to modify this PR.

**Comment by meomancer** [comment]
i see, that make sense @boney-bun 
if there will be others VAT changes, we need to make history for that
i think, simple change like this is enough

for me LGTM

**Review by meomancer — APPROVED** [review]

**Comment by boney-bun** [comment]
thanks for your feedback @meomancer 
i'm going to merge and bring it to test server

**Comment by gubuntu** [comment]
@boney-bun is this deployed on testing? Please send me testing admin credentials so I can see, as it should go into production asap

**Comment by boney-bun** [comment]
@gubuntu yes, this is deployed on testing already.
the same credentials on production should work on testing now.

**Comment by gubuntu** [comment]
LGTM, release to production

**Pull request merged**